### ISSUE TITLE
open bookmarks in a new tab

### ISSUE THREAD

**Issue by chrispickard** [cmcaine/tridactyl#902]
Hey, thanks for tridactyl! Is there a way to open bookmarks in a new tab (essentially combine `:bmarks` with `:tabopen`)?

**Comment by bovine3dom** [comment]
No, there isn't. I'd happily forgotten the existence of `bmarks`. It's one of the most awful hacks left in Tridactyl. You *could* just copy and paste this file https://github.com/cmcaine/tridactyl/blob/f36b012af45dd68f383f5727c914c9b339573b70/src/completions/Bmark.ts#L95 and change the bmarks to bmarkst and open to `tabopen`... but I'd much rather we just rewrite completions to be quick and support multiple sources.

Which I assumed was what #895 was about.

**Comment by glacambre** [comment]
>Which I assumed was what #895 was about.

This isn't really what #895 was about. I just wanted to have excmd completion (as in completing just the name of the excmd) because I thought it was cool :D 
I think writing all these completion sources helped me identify what we might need for a "proper" completion mechanism though and I'll start thinking about how to implement it. This is going to take a very long while though, I'm going to be very busy with other things for a few weeks.
